### DISCUSSION TITLE
是否有数据库脚本版本管理支持的计划

### DISCUSSION THREAD

**Discussion by TonyJiangWJ** [ccmjga/mjga-scaffold#9] [Ideas]
目前数据库脚本只在第一次创建容器并运行的时候才会执行，后续增量脚本都需要手动执行。作者有没有计划接入数据库脚本版本管理的框架，用以实现自动化执行增量脚本的功能。

**Comment by ccmjga** [comment]
这个功能可以做，前段时间调研过，但是之前因为其他一些原因搁置了。
待我把手里的事情完成后我继续研究一下，功能肯定是可以实现的，只是有些利弊需要权衡。

**Comment by ccmjga** [comment]
2.8 版本已经正式开启公测。这个版本增加了一个很重要的功能：集成了开箱即用的 DDL 和 Flyway。

对于 database first 这个模板来说，DDL 和 Flyway 是它的最后一块拼图，是以数据库为一等公民架构思想的重要表现媒介。现如今有了他们的集成，dbfirst 模板才真正进化到了完全体形态。
 
得益于这两个工具的集成，自动化执行增量脚本的功能也能够实现了。op 需要的话可以试用并测试，如果遇到 bug 或者在使用中的其他建议，请告诉我。
